guide/daily-spending-wallet/requesting/receiving.md

```diff
@@ -0,0 +1,181 @@
+
+{% include tip/open.html color="blue" icon="info" label="Experimental options" %}
+
+Experimental options such as [Peerswap](https://www.peerswap.dev/) and [Collaborative funding](https://bitcoinops.org/en/topics/dual-funding/) may be more trustless options that could be used to receive on-chain payments whilst maintaining a single lightning balance.
+
+{% include tip/close.html %}
+
+### Refunds
+
+There may be scenarios where a submarine swap is not completed successfully, such as when the wallets LSP is offline. In these cases, the users will need to claim the on-chain funds that failed to be swapped by sending them to a on-chain address. 
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   layout = "shadow"
+   caption = "If a submarine swap fails, notify the user that they need to claim the refunded payment."
+   alt-text = "Screen showing a warning error notification prompting the user to claim a refund of their failed submarine swap."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   layout = "shadow"
+   caption = "The activity page is another place where you can indicate to a user that they need to claim their refund."
+   alt-text = "Screen showing the activity menu with an entry for the user to claim their refunded payment."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   layout = "shadow"
+   caption = "Make it simple for a user to claim their refund."
+   alt-text = "Screen showing a form to claim the refund to an on-chain address."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+</div>
+
+## Received payment
+
+Once a payment is received it should be clearly indicated to the user and their balance should be updated. Ensure fees charged, if any, are clear to the user when viewing the payments details. Find more information on presenting a payment history on our [activity]({{ '/guide/daily-spending-wallet/activity/'| relative_url }}) page. 
+
+The option to save and share a receipt should be readily available.
```

Comment:
I don't know what readily available means, just available maybe?

Reply:
Just means easily, but agree it's not necessary will remove